java-speech/google-cloud-speech/src/main/java/com/google/cloud/speech/v1/stub/HttpJsonSpeechStub.java

```diff
@@ -212,7 +409,7 @@ protected HttpJsonSpeechStub(
             longRunningRecognizeTransportSettings,
             settings.longRunningRecognizeOperationSettings(),
             clientContext,
-            httpJsonOperationsStub);
+            longRunningClient);
```

Comment:
This seems to be the crux of this approach. The minimum we can do to make it work is to create a `longRunningClient` with a `getOperationCallable` and the respective `getOperationMethodDescriptor`

Reply:
I believe so. Assuming that the service.yaml file doesn't have any other Operation RPCs listed in the api.name field (which would mark those RPCs as supported), we can just create a LongRunningClient with the GetOperationCallable and DeleteOperation + CancelOperation callables can be null.